code/week6/main.cpp

```diff
@@ -0,0 +1,58 @@
+#include <opencv2/opencv.hpp>
```

Comment:
I think this is the first time we're using Cmake in the course, can you add a list of commands to run to this file (or add a makefile?)

```
mkdir build
cd build
cmake ../
make -j
./findBlue
```